other/nightly/install.sh

```
@@ -288,169 +431,245 @@ if [ -x "$(command -v snap)" ]; then
+        systemctl start docker >/dev/null 2>&1
+        systemctl enable docker >/dev/null 2>&1
+        if ! [ -x "$(command -v docker)" ]; then
+            echo " - Failed to install Docker with dnf. Try to install it manually."
+            echo "   Please visit https://www.cyberciti.biz/faq/how-to-install-docker-on-amazon-linux-2/ for more information."
+            exit 1
+        fi
+        ;;
+    "centos" | "fedora" | "rhel")
+        if [ -x "$(command -v dnf5)" ]; then
+            # dnf5 is available
+            dnf config-manager addrepo --from-repofile=https://download.docker.com/linux/$OS_TYPE/docker-ce.repo --overwrite >/dev/null 2>&1
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)